src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -172,3 +178,23 @@ export default class SourceTreeItem extends Component<Props, State> {
     );
   }
 }
+
+function hasMatchingGeneratedSource(state, source: ?Source) {
+  if (!source) {
+    return false;
+  }
+
+  const sources = getSourcesByURL(state, source.url);
+  return isOriginalId(source.id) && sources.length > 1;
+}
+
+const mapStateToProps = (state, props) => {
+  return {
+    isDup: hasMatchingGeneratedSource(state, props.source)
```

Comment:
what's a good name for this property?

Reply:
`hasMatchingGeneratedSource` seems a reasonable property name to me :)